pipeline-braille-scripts/epub3-to-pef/src/main/resources/xml/xproc/epub3-to-pef.convert.xpl

```diff
@@ -0,0 +1,300 @@
+        <p:when test="not(/*/*[@name='toc-depth']/@value = '0')">
+            <px:fileset-add-tempfile media-type="text/css" suffix=".css">
+                <p:input port="source">
+                    <p:inline>
+                        <c:data>#generated-toc {
+  flow: document-toc;
+  display: -obfl-toc;
+  -obfl-toc-range: document;
+}
+
+#generated-toc::duplicate {
+  flow: volume-toc;
+  display: -obfl-toc;
+  -obfl-toc-range: volume;
+}
+</c:data>
+                    </p:inline>
+                </p:input>
+            </px:fileset-add-tempfile>
+        </p:when>
+        <p:otherwise>
+            <p:identity/>
+        </p:otherwise>
+    </p:choose>
+    <p:identity name="generated-css"/>
+    
+    <!-- Load XHTML documents in spine order. -->
+    <px:fileset-load media-types="application/oebps-package+xml application/xhtml+xml">
+        <p:input port="fileset">
+            <p:pipe port="result" step="opf-fileset"/>
+        </p:input>
+        <p:input port="in-memory">
+            <p:pipe port="in-memory.in" step="main"/>
+        </p:input>
+    </px:fileset-load>
+    <p:for-each>
+        <p:add-attribute match="/*" attribute-name="xml:base">
+            <p:with-option name="attribute-value" select="base-uri(/*)"/>
+        </p:add-attribute>
+    </p:for-each>
+    <p:wrap-sequence wrapper="wrapper"/>
+    <p:xslt>
+        <p:input port="parameters">
+            <p:empty/>
+        </p:input>
+        <p:input port="stylesheet">
+            <p:document href="../xslt/get-epub3-spine.xsl"/>
+        </p:input>
+    </p:xslt>
+    <p:filter select="/*/*"/>
+    
+    <!-- In case there exists any CSS in the EPUB already, and $apply-document-specific-stylesheets = 'true',  then inline that CSS. -->
+    <px:message message="Processing CSS that is already present in the EPUB"/>
+    <p:for-each>
+        <p:add-attribute match="/*" attribute-name="xml:base">
+            <p:with-option name="attribute-value" select="base-uri(/*)"/>
+        </p:add-attribute>
+        
+        <px:message severity="DEBUG">
+            <p:with-option name="message" select="concat('Deleting CSS that is not for embossed media from ',replace(base-uri(/*),'.*/',''),'')"/>
+        </px:message>
+        <p:delete match="//@style | //html:link[@rel='stylesheet' and not(string(@media)='embossed')] | //html:style[not(string(@media)='embossed')]"/>
+        
+        <p:choose>
+            <p:xpath-context>
+                <p:pipe step="parameters" port="result"/>
+            </p:xpath-context>
+            <p:when test="/*/*[@name='apply-document-specific-stylesheets']/@value = 'true'">
+                <px:message severity="DEBUG">
+                    <p:with-option name="message" select="concat('Inlining document-specific CSS for ',replace(base-uri(/*),'.*/',''),'')"/>
+                </px:message>
+                <p:group>
+                    <!-- <link> not supported in css:inline so we provide the URIs to them explicitly -->
```

Comment:
I'm pretty sure `link` is supported. Did your example have `rel="stylesheet"` and `type="text/css"`?